doc/api/errors.md

```diff
@@ -2065,6 +2078,13 @@ instance.setEncoding('utf8');
 An attempt has been made to create a string larger than the maximum allowed
 size.
 
+<a id="ERR_TTY_WRITABLE_NOT_READABLE"></a>
+### ERR_TTY_WRITABLE_NOT_READABLE
```

Comment:
Nit: `###` -> `####` for level consistency)

Reply:
Done … this seems like something that our linter could maybe catch as well?

Reply:
It seems, in this case, it will not be clear for a linter if this is a new level or just a typo.